tests/phpunit/integration/Core/Consent_Mode/REST_Consent_Mode_ControllerTest.php

```diff
@@ -323,92 +323,92 @@ public function test_get_ads_measurement_status() {
 
 	public function test_get_ads_measurement_status__ads_module_connected() {
 		$this->setup_rest();
-		// Setup the site and admin user to make a successful REST request.
 		$this->grant_manage_options_permission();
 
-		$this->force_module_connection( Ads::MODULE_SLUG );
+		add_filter(
+			'googlesitekit_ads_measurement_connection_checks',
```

Comment:
Before testing with filters, its important to always make sure they are cleared of any hooked callbacks to ensure the test is using the expected setup by registering the class afterwards. This way we know only the test/class under test is responsible for the outcome.